lib/telephony/pinpoint/voice_sender.rb

```diff
@@ -35,6 +38,7 @@ def send(message:, to:, country_code:, otp: nil)
             extra: {
               message_id: response.message_id,
               duration_ms: Util.duration_ms(start: start, finish: finish),
+              origination_phone_number: origination_phone_number,
```

Comment:
I figured we'd just log this whole number, since it's Login.gov's, it's not customer PII? Goes against my instincts but I think it's fine. Much easier to debug this way.

Reply:
Yep, this is something we can see in the console anyway and we log similar things for SMS, so this should be fine